pytket/binders/mapping.cpp

```diff
@@ -149,7 +149,10 @@ PYBIND11_MODULE(mapping, m) {
           "qubits.\n\n:param circuit: pytket circuit to be mapped"
           "\n:param routing_methods: Ranked methods to use for routing "
           "subcircuits. In given order, each method is sequentially checked "
-          "for viability, with the first viable method being used.",
-          py::arg("circuit"), py::arg("routing_methods"));
+          "for viability, with the first viable method being used."
+          "\n:param label_single_qubits: will not label qubits without gates "
```

Comment:
Maybe `label_isolated_qubits` would be a better name?

Reply:
done